account_spread_cost_revenue/models/account_spread.py

```diff
@@ -44,6 +44,10 @@ class AccountSpread(models.Model):
         default='month',
         help="Period length for the entries",
         required=True)
+    use_invoice_line_account = fields.Boolean(
+        string="Use invoice line's account",
+        default=False,
```

Comment:
Could you remove the `default=False` here and in the template? By default the value of a boolean is already false.
Moreover, if you remove `default=False`, you avoid conflicts in case a user wants to manually set a different default by changing the configuration of the field.